graphdatascience/tests/integration/test_ogb_loader.py

```diff
@@ -0,0 +1,97 @@
+import pytest
+
+from graphdatascience.graph_data_science import GraphDataScience
+from graphdatascience.server_version.server_version import ServerVersion
+
+
+@pytest.mark.enterprise
+@pytest.mark.ogb
+@pytest.mark.compatible_with(min_inclusive=ServerVersion(2, 1, 0))
+def test_graph_load_ogbn_arxiv(gds: GraphDataScience) -> None:
+    # ogbn-arxiv is a homogeneous dataset for node classification
+
+    G = gds.graph.ogbn.load("ogbn-arxiv")
+
+    assert G.name() == "ogbn-arxiv"
+    assert G.node_count() == 169_343
+    assert set(G.node_labels()) == {"Train", "Test", "Valid"}
+    for lbl in G.node_labels():
+        assert set(G.node_properties()[lbl]) == {"features", "classLabel"}  # type: ignore
+    assert G.relationship_count() == 1_166_243
+    assert G.relationship_types() == ["R"]
+    assert G.relationship_properties()["R"] == []  # type: ignore
+
+    G.drop()
+
+
+@pytest.mark.enterprise
+@pytest.mark.ogb
```

Comment:
they are not needed to be marked as enterprise are they?

Reply:
True! Not anymore. But I don't think it's gonna be very practical to run on CE as it's probably gonna crash Neo4j unless they have a big max heap